src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public void validate(String password, String repeatPassword) {
-        //write your code here
+    public void validate(String password, String repeatPassword)
+            throws PasswordValidationException {
```

Comment:
you can inline `throws`
```suggestion
    public void validate(String password, String repeatPassword) throws PasswordValidationException {
```